### ISSUE TITLE
1202 Layer Net

### ISSUE THREAD

**Issue by yueatsprograms** [facebook/fb.resnet.torch#20]
Hi,

Have you ever tried to train the 1202 layer net on CIFAR-10?
Using the current code, this large net (with batch size 128 on 2 GPUs) doesn't seem to converge. Even if I lower the learning rate to 0.01 for the first 5 epochs, it doesn't seem to converge.
Would really appreciate anyone's input on this! Thanks!

Yu

**Comment by colesbury** [comment]
It converges for me when I set the learning rate to 0.01 for the first epoch. You''ll need to modify Trainer:learningRate to get that behavior:

https://github.com/facebook/fb.resnet.torch/blob/master/train.lua#L169

**Comment by yueatsprograms** [comment]
Right, so this is what actually happens with this schedule if training goes on... Have you tried beyond 30 epochs?
<img width="654" alt="screen shot 2016-03-05 at 6 02 01 pm" src="https://cloud.githubusercontent.com/assets/5546216/13551907/803b830c-e317-11e5-82a1-eeb1faef04f2.png">

**Comment by colesbury** [comment]
Error = 7.3% at epoch 85. Here's a test error curve for ResNets 20,32,44,56,110,1202.

ResNet 1202 is in yellow.

<img width="831" alt="resnet cifar10" src="https://cloud.githubusercontent.com/assets/655866/13551931/58aea660-e318-11e5-9c9a-73cf2d59d519.png">

The jump in your error rate around epoch 28 is really strange. Can you post your code?
